piton/design/xilinx/genesys2/constraints.xdc

```diff
@@ -58,6 +58,9 @@ set_false_path -from [get_ports trst_ni]
 set_max_delay -datapath_only -from [get_clocks -include_generated_clocks chipset_clk_clk_mmcm] -to [get_clocks tck_i] 15.000
 set_max_delay -datapath_only -from [get_clocks tck_i] -to [get_clocks -include_generated_clocks chipset_clk_clk_mmcm] 15.000
 
+# Retiming
+set_property BLOCK_SYNTH.RETIMING 1 [get_cells -hierarchical *pipe_fma*]
+
```

Comment:
This didn't end up helping much, right? Should we keep this out and wait for the manually retimed FPU?